mix.exs

```diff
@@ -46,7 +46,7 @@ defmodule Paginator.Mixfile do
       {:ex_doc, "~> 0.30", only: :dev, runtime: false},
       {:ex_machina, "~> 2.1", only: :test},
       {:postgrex, "~> 0.13", optional: true},
-      {:plug_crypto, "~> 2.0.0"}
+      {:plug_crypto, "~> 1.2"}
```

Comment:
Hi @btkostner, I've downgraded this back to `1.2` as the `2.0.0` version is not compatible with Phoenix 1.6 which is used by inventory. Tested it on staging and seems to be working fine

Reply:
This should be fine though you'll hit OTP 26 errors in your tests

Reply:
Thanks! The tests ran fine for me.